Packs/FiltersAndTransformers/TestPlaybooks/playbook-TestIsNotInCidrRanges.yml

```diff
@@ -0,0 +1,273 @@
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "15":
+    id: "15"
+    taskid: 556ec2a7-24ee-434e-81db-bbf31aaeea9b
+    type: condition
+    task:
+      id: 556ec2a7-24ee-434e-81db-bbf31aaeea9b
+      version: -1
+      name: Verify Output Is True
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      "yes":
+      - "8"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              simple: ${scriptResult}
+            iscontext: true
+          right:
+            value:
+              simple: "True"
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 895
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "16":
+    id: "16"
+    taskid: 5b3c401e-4193-4365-8ff8-2bb05367ec82
+    type: regular
+    task:
+      id: 5b3c401e-4193-4365-8ff8-2bb05367ec82
+      version: -1
+      name: Set IPs
+      description: Set a value in context under the key you entered.
+      scriptName: Set
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "17"
+    scriptarguments:
+      key:
+        simple: ips
+      value:
+        simple: '["192.168.1.10","11.1.1.1"]'
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 370
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "17":
+    id: "17"
+    taskid: b6529f09-5ba8-4e77-8f86-8a54a45c965e
+    type: regular
+    task:
+      id: b6529f09-5ba8-4e77-8f86-8a54a45c965e
+      version: -1
+      name: Set CIDR ranges
+      description: Set a value in context under the key you entered.
+      scriptName: Set
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "14"
+    scriptarguments:
+      key:
+        simple: cidrs
+      value:
+        simple: '["192.168.1.0/24","192.168.1.0/287"]'
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 545
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+view: |-
+  {
+    "linkLabelsPosition": {},
+    "paper": {
+      "dimensions": {
+        "height": 1085,
+        "width": 380,
+        "x": 50,
+        "y": 50
+      }
+    }
+  }
+inputs: []
+outputs: []
+fromversion: 6.10.0
```

Comment:
Why is the tpb from version 6.10 and the yml from version 5.0.0?

Reply:
done